pkg/k8s/service.go

```diff
@@ -91,6 +91,39 @@ func isValidServiceFrontendIP(netIP net.IP) bool {
 	return false
 }
 
+type exposureType struct {
+	isSet   bool
```

Comment:
This field is unnecessary see [1] and [2]. We can use the pointer of the struct itself to check if it's set or not.